java/pom.xml.in

```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.github.vowpalwabbit</groupId>
     <artifactId>vw-jni</artifactId>
-    <version>@PACKAGE_VERSION@-SNAPSHOT</version>
+    <version>@PACKAGE_VERSION@.2</version>
```

Comment:
Should this .2 be here?

Reply:
yep, that's the .jar sub-version. e.g. 8.7.0.1 vs 8.7.0.2